netlify.toml

```diff
@@ -8,6 +8,11 @@
   to = "/index.html"
   status = 200
 
+[[redirects]]
```

Comment:
We want to be careful with netlify -> they changed their client URL settings so you get a unique url: (here's a recent lobby I made: https://6254700f190d1600abe463cf--df-prod.netlify.app/play/0x832dde02116c8f6b5e47cf572f45a62abafb2831)

Not sure how deep the changes are, but need to make sure we don't incorporate any of them.